Screens/ProgramsInfo.js

```diff
@@ -0,0 +1,183 @@
+import * as React from "react";
+import {
+  StyleSheet,
+  Text,
+  View,
+  Button,
+  Alert,
+  Pressable,
+  Image,
+  SafeAreaView,
+  ScrollView,
+} from "react-native";
+
+import { AntDesign } from "@expo/vector-icons";
+import { Entypo } from "@expo/vector-icons";
+import { db } from "../firebase-config.js";
+import { getDatabase, ref, get, child } from "firebase/database";
+
+const programsRef = ref(getDatabase(), "programs");
```

Comment:
why does this screen exist if course.js also exists? we should only have one. and why does it access the database? we were passing info with navigation function so that we don't have to access the database again.

Reply:
These exist when I download the code, and I didn't add them and used them, just keep it now and I will delete them later.